core.js

```diff
@@ -0,0 +1,70 @@
+const _ = require("lodash");
+const helpers = require("./helpers");
+const autocomplete = require("./autocomplete");
+const consts = require("./consts.json");
+
+function getServiceInstance(
```

Comment:
Please order the functions from the outermost to innermost, maintaining the export order.